src/aws/osml/tile_server/viewpoint/viewpoint_id/image/info.py

```diff
@@ -0,0 +1,30 @@
+#  Copyright 2024 Amazon.com, Inc. or its affiliates.
+
+from typing import Annotated
+
+from fastapi import APIRouter, Depends, status
+from starlette.responses import FileResponse
+
+from aws.osml.tile_server.app_config import ServerConfig
+from aws.osml.tile_server.services import ViewpointApiNames, get_aws_services, validate_viewpoint_status
+
+info_router = APIRouter(
+    prefix="/info",
+    dependencies=[],
+    responses={status.HTTP_404_NOT_FOUND: {"description": "Not found!"}},
+)
+
+
+@info_router.get("")
+def get_image_info(viewpoint_id: str, aws: Annotated[get_aws_services, Depends()]) -> FileResponse:
+    """
+    Get a sample GeoJSON feature that represents the extent / boundary of this image in the world.
+
+    :param viewpoint_id: Unique viewpoint id to get from the table.
```

Comment:
Update params docstring and return docstring. Please go through all the files you have created and validate if those functions docstring have been updated.